components/developer/gcc-10/gcc-10.p5m

```diff
@@ -908,29 +908,29 @@ file path=usr/gcc/10/include/c++/$(COMPONENT_VERSION)/$(GNU_TRIPLET)/bits/stdc++
 file path=usr/gcc/10/include/c++/$(COMPONENT_VERSION)/$(GNU_TRIPLET)/bits/stdtr1c++.h
 file path=usr/gcc/10/include/c++/$(COMPONENT_VERSION)/$(GNU_TRIPLET)/bits/time_members.h
 file path=usr/gcc/10/include/c++/$(COMPONENT_VERSION)/$(GNU_TRIPLET)/ext/opt_random.h
```

Comment:
I wonder whether the files about should also have a variant.arch, no?

Reply:
Probably not since @tsoome did not need to flag them as i386-specific nor added them for the sparc build.